python/ray/tests/test_advanced_8.py

```diff
@@ -192,6 +192,18 @@ def test_ray_resources_environment_variable(ray_start_cluster):
     assert cluster_resources["CPU"] == 3
 
 
+def test_ray_labels_environment_variables(shutdown_only):
```

Comment:
Could we also add a small test for https://github.com/ray-project/ray/blob/0e0be3a86e3a608bf9418bbedce5d2e74cf2953a/python/ray/util/state/common.py#L511-L512 ?